Modules/parsermodule.c

```diff
@@ -1158,6 +1158,12 @@ PyInit_parser(void)
 {
     PyObject *module, *copyreg;
 
+    if (PyErr_WarnEx(PyExc_DeprecationWarning,
+            "The parser module is deprecated and will be removed "
+            "in future versions of Python", 7) != 0) {
```

Comment:
7 is the magic number to skip all frozen importlib (tested in `test_deprecation_message`). :upside_down_face: